static/app/components/events/interfaces/frame/stacktraceLink.tsx

```diff
@@ -186,7 +186,7 @@ function CodecovPrompt({organization}: {organization: Organization}) {
 
   return (
     <OpenInLink
-      href="https://about.codecov.io/sign-up-sentry-codecov/"
+      href="https://about.codecov.io/sign-up-sentry-codecov/?utm_medium=referral&utm_source=sentry-app-stacktrace-link&utm_campaign=sentry-codecov&utm_department=marketing"
```

Comment:
@scttcper where is the `state` parameter? We have to load that asynchronously from the server.